target/linux/x86/64/config-5.4

```diff
@@ -311,6 +313,7 @@ CONFIG_ND_CLAIM=y
 CONFIG_NEED_DMA_MAP_STATE=y
 CONFIG_NET_FAILOVER=y
 CONFIG_NET_FLOW_LIMIT=y
+CONFIG_NET_VENDOR_AMAZON=y
```

Comment:
Nevertheless, this is already enabled in the default configuration

Reply:
good catch, that option enables Amazon network drivers for T3 cloud instance, so it imho should be removed from default config and kept only here.

I can add a commit to drop that from default config, is that OK?

Reply:
All vendors are disabled by default in the default configuration, as none of them change the artifacts generated but simplify the module packaging.

Reply:
seem to me they are all enabled by default in default config https://github.com/openwrt/openwrt/blob/41b096b44861cf630fff9a991ebc9b4b006ce7ac/target/linux/generic/config-5.4#L3539

But anyway, if the consensus is not touching that I'm just dropping the CONFIG_NET_VENDOR_AMAZON=y  from this patch.